test/hosting/NuGet.Config

```diff
@@ -2,6 +2,6 @@
 <configuration>
```

Comment:
I think this file can be removed. It can just use the nuget.config file at the repo root.